### PR TITLE
Fix for WFWIP-216 and WFWIP-217, fully separate galleon and startup s…

### PR DESCRIPTION
**Pull request by jfdenise**
…ettings from user settings
- server and tools are configuring JBoss modules to retrieve settings.xml thanks to -Djboss.modules.settings.xml.url system property.
- galleon provisioning relies on its own settings.xml
- user settings are un-touched.
- In addition cleanup un-used modules.

### CODE REVIEW

File: jboss/container/wildfly/standalone-conf/java/added/standalone.conf [hunk 1]
```diff
@@ -1,15 +0,0 @@
-
-source /usr/local/dynamic-resources/dynamic_resources.sh
-export GC_MAX_METASPACE_SIZE=${GC_MAX_METASPACE_SIZE:-256}
-JAVA_OPTS="$(adjust_java_options ${JAVA_OPTS})"
-
-# Make sure that we use /dev/urandom (CLOUD-422)
-JAVA_OPTS="${JAVA_OPTS} -Djava.security.egd=file:/dev/./urandom"
```
Comment (luck3y) [inline jboss/container/wildfly/standalone-conf/java/added/standalone.conf]: Did -Djava.security.egd=file:/dev/./urandom get moved somewhere else, or is it not needed anymore?

File: jboss/container/wildfly/standalone-conf/java/added/standalone.conf [hunk 2]
```diff
@@ -1,15 +0,0 @@
...
-
-# White list packages for use in ObjectMessages: CLOUD-703
-if [ -n "$MQ_SERIALIZABLE_PACKAGES" ]; then
-  JAVA_OPTS="${JAVA_OPTS} -Dorg.apache.activemq.SERIALIZABLE_PACKAGES=${MQ_SERIALIZABLE_PACKAGES}"
```
Comment (luck3y) [inline jboss/container/wildfly/standalone-conf/java/added/standalone.conf]: Same question as above, but regarding ${MQ_SERIALIZABLE_PACKAGES}

File: jboss/container/wildfly/standalone-conf/java/added/standalone.conf (hunk 1 above)
Comment (jfdenise) [reply jboss/container/wildfly/standalone-conf/java/added/standalone.conf]: yes, all was moved some time ago....https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/galleon/fp-content/java/added/src/main/resources/packages/wildfly.s2i.java/content/bin/java-conf.conf

File: jboss/container/wildfly/standalone-conf/java/added/standalone.conf (hunk 2 above)
Comment (jfdenise) [reply jboss/container/wildfly/standalone-conf/java/added/standalone.conf]: Same reply https://github.com/wildfly/wildfly-cekit-modules/blob/master/jboss/container/wildfly/galleon/fp-content/java/added/src/main/resources/packages/wildfly.s2i.java/content/bin/java-conf.conf

**Comment by jbliznak** [comment]
What is expected content of settings.xml in time of mvn app build when user is doing S2I + providing own settings.xml?

**Comment by jfdenise** [comment]
@jbliznak , during s2i, if user provides his settings.xml, that is what we will use for the mvn build of the app. Galleon relies on its own settings.xml that is now fully un-coupled.

File: jboss/container/wildfly/standalone-conf/java/added/standalone.conf (hunk 1 above)
Comment (luck3y) [reply jboss/container/wildfly/standalone-conf/java/added/standalone.conf]: Got it, thanks!

**Pull request merged**